### PR TITLE
feat(columbus): add "service" field to columbus payload

### PR DESCRIPTION
**Pull request by StewartJingga**

### CODE REVIEW

File: models/metadata.go [hunk 1]
```diff
@@ -9,4 +9,5 @@ import (
 type Metadata interface {
 	GetResource() *common.Resource
 	GetProperties() *facets.Properties
+	GetLineage() *facets.Lineage
```
Comment (ravisuhag) [inline models/metadata.go:12]: @StewartJingga This would not be applicable to every resource right? Like user.

Comment (StewartJingga) [reply models/metadata.go:12]: yes, but right now it is the only way Columbus sink is able to fetch lineage from our varies models.

The other way is to typecasting for each types which im not sure is scalable.

Comment (StewartJingga) [reply models/metadata.go:12]: @ravisuhag another approach is to have another interface for lineage and typecheck for that interface in Columbus.
```
type HasLineage interface {
  GetLineage() *facets.Lineage
}
```

Comment (ravisuhag) [reply models/metadata.go:12]: I think let's create another interface for lineage check instead of expecting all models to have it.

**Review by ravisuhag — APPROVED** [review]
LGTM

**Pull request merged**